### ISSUE TITLE
Build Image on Host

### ISSUE THREAD

**Issue by williballenthin** [fireeye/capa-rules#345]
https://attack.mitre.org/techniques/T1612

**Comment by williballenthin** [comment]
`docker build`

**Comment by williballenthin** [comment]
https://www.virustotal.com/gui/search/(type%253Apedll%2520OR%2520type%253Apeexe)%2520AND%2520%2522docker%2520build%2522/files

![image](https://user-images.githubusercontent.com/156560/116719955-e0687f80-a998-11eb-911f-c05cf6e8f1c2.png)

```
4bb503a7e942a0ab6350bd34c937d0fd
6deaf06d63b76f90e529e146609bd85e
98cda0cb694f8687d52abfdb2cf515c8
1615fc0c7a1857b3b8307234e12d6eca
dbdf8a8b19fd2008be63dee1800d6af1
```

**Comment by williballenthin** [comment]
closed in #365
